scripting/z_esc_reference.rst

```diff
@@ -76,9 +76,67 @@ Events
 All ESC scripts are divided into a series of events which in turn run
 commands or dialogs.
 
-An event has a name and the prefix ``:``, like this:
+To use an event in your script, specify the name of the event preceded by a 
+colon. All commands following the event identifier are considered part of that
+event until another event is defined in the same script file.
+
+.. code-block::
+
+   :push
+   say player "I pushed the door."
+   say player "It didn't do anything."
+
+   # A new ("pull") event starts here
+   :pull
+   say player "The door is now open"
+
+In-built events
+^^^^^^^^^^^^^^^
+
+Some events are hard-coded into Escoria. The ones that are 
```

Comment:
I see this is a specific version of the template from above. Please copy the changes from that section into this one.

Reply:
Not sure what you mean here, these 4 lines aren't a template?

Reply:
As discussed, not changing ./getting_started/z_escoria_settings.rst as it should be generated from this template.